ykcapi/src/lib.rs

```diff
@@ -170,6 +172,48 @@ pub extern "C" fn yk_stopgap(
     process::exit(0);
 }
 
+/// The `__llvm__deoptimize()` function required by `llvm.experimental.deoptimize` intrinsic, that
+/// we use for exiting to the stop-gap interpreter on guard failure.
+#[cfg(target_arch = "x86_64")]
```

Comment:
It's just occurred to me that we should have a `compile_error!` for other arches. Agree?

Reply:
Fixed in eb0bc4e

Reply:
Good idea, as without that people will get a "symbol not found" (which is better than nothing, but not always the most helpful message!).